docs-src/httproute.md

```diff
@@ -0,0 +1,303 @@
+# HTTPRoute
+
+[HTTPRoute][httproute] is a Service APIs type for specifying routing behavior of HTTP requests from a Gateway listener
+to an API object, i.e. Service.
+
+## Spec
+
+The specification of an HTTPRoute consists of:
+
+- [Gateways][gateways]- Define which Gateways can use this HTTPRoute.
+- [Hostnames][hostname] (optional)- Define a list of hostnames to use for matching the Host header of HTTP requests.
+- [TLS][tls-config] (optional)- Defines the TLS certificate to use for Hostnames defined in this Route.
+- [Rules][httprouterule]- Define a list of rules to perform actions against matching HTTP requests. Each rule consists
+  of [matches][matches], [filters][filters] (optional), and [forwardTo][forwardto] (optional) fields.
+
+The following illustrates an HTTPRoute that sends all traffic to one Service:
+![httproute-basic-example](httproute-basic-example.svg)
+
+### Gateways
+
+Gateways define which Gateways can use the HTTPRoute. If unspecified, `gateways` defaults to `allow: SameNamespace`
+which allows all Gateways in the HTTPRoute's namespace.
+
+The following example allows Gateways from namespace "httproute-ns-example":
+```yaml
+kind: HTTPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: httproute-example
+  namespace: httproute-ns-example
+spec:
+  gateways:
+    allow: SameNamespace
+```
+
+Possible values for `allow` are:
+
+- `All`: Gateways in any namespace can use this route.
+- `FromList`: Only Gateways specified in `gatewayRefs` may use this route.
+- `SameNamespace` (default): Only Gateways in the same namespace may use this route.
+	
+If `allow` results in preventing the selection of an HTTPRoute by a Gateway, an “Admitted: false” condition must be set
+on the Gateway for this Route.
```

Comment:
Nit: Instead of "must" maybe we should use "will". This kind of doc seems like it should be more targeted towards users than implementers.